### DISCUSSION TITLE
How long do you train the model ?

### DISCUSSION THREAD

**Discussion by cjdjr** [STVIR/pysot#321] [General]
I find it costs lost of time to train a model using four 2080ti.
How long do you train the model "siamrpn_r50_l234_dwxcorr_otb_8gpu"? And what type of gpu you used to train the model?

**Comment by ZhiyuanChen** [comment]
How long do you mean by "lots of time"?

**Comment by cjdjr** [comment]
> How long do you mean by "lots of time"?

7 days.

**Comment by MARMOTatZJU** [comment]
@cjdjr  and what about the results?

**Comment by zyh0904** [comment]
I only have one GPU, how can i train the model ????? how to change the code ??

**Comment by ZhiyuanChen** [comment]
> > How long do you mean by "lots of time"?
> 
> 7 days.

Wow, that's a lot more than expected. 
How many data did you feed to the model? And how many epoch did you trained? 
What is the batch size and how long does it take to train a step?

**Comment by ZhiyuanChen** [comment]
Since no further question is asked, I'm closing this issue for now. 
Please create a new issue should you have other questions.
